src/components/GlobalSearch.jsx

```diff
@@ -8,7 +8,7 @@ function GlobalSearch() {
           <input type="search" id="default-search" className="placeholder-text-bold w-full p-4 pl-10 text-sm text-gray-900 border border-borders-100 rounded bg-gray-50 focus:ring-blue-500 focus:border-blue-500" placeholder="Search by Style or Title or Brand" required/>
       </div>
       <div className="m-0">
-      <button className="text-white bg-black py-3.5 px-4 rounded ml-2">Search</button>
+      <button className="text-white bg-black py-3.5 px-4 rounded ml-2">{serachString}</button>
```

Comment:
why is button still not a component yet ?

Reply:
done created a separate component